app/src/androidTest/java/com/example/android/courtcounter/Tests/courtCounterTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.android.courtcounter.testFolder
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.example.android.courtcounter.MainActivity
+import com.example.android.courtcounter.R
+import com.example.android.courtcounter.Utils.BaseTest
+import org.hamcrest.Matchers.allOf
+import org.junit.Test
+import org.junit.runner.RunWith
+
+//type out the run with line and it will import the required parts
+@RunWith(AndroidJUnit4::class)
+class myFirstTest: BaseTest() {
```

Comment:
Its up to you what one you go with, but I'd pick one standard for class names, leading capital or camel case, and be consistent throughout